rsearch/conductor.go

```diff
@@ -0,0 +1,43 @@
+package rsearch
+
+func manageResources(ns Event, terminators map[string]chan Done, config Config, out chan Event) {
+	uid := ns.Object.Metadata.Uid
+	if ns.Type == "ADDED" {
+		done := make(chan Done)
+		terminators[uid] = done
+		ns.Object.Produce(out, terminators[uid], config)
+	} else if ns.Type == "DELETED" {
+		close(terminators[uid])
```

Comment:
Shouldn't really happen but we will blow up in this case. However, spitting out readable message could be helpful.